src/components/MovieDetails/MovieDetails.js

```diff
@@ -30,25 +30,27 @@ const MovieDetails = () => {
           onClick={handleSearchMovie}
         />
       </div>
-      <div className="movie-details__main">
-        <img src={movie.poster_path} alt="poster" />
-        <div className="movie-details__all">
-          <div className="movie-details__first-row">
-            <div className="movie-details__title">{movie.title}</div>
-            <div className="movie-details__rate">
+      <div className="details__content">
+        <img
+          className="details-content__image"
+          src={movie.poster_path}
+          alt="poster"
+        />
+        <div className="details">
```

Comment:
See, above, you have "details" and here you have "details" again, which is which?

I'd rename the component to a `Movie` and use here a `movie__details` class